build/constants.yaml

```diff
@@ -51,7 +51,7 @@
     python-base-tag: 20190610.2
     pip-version: 19.0.3
     python27-version: 2.7.16
-    python36-version: 3.6.8
+    python36-version: 3.6.9
```

Comment:
run the constants generator?

Reply:
yeah, not done with this PR yet, should have marked it as WIP